src/query.rs

```diff
@@ -407,6 +407,10 @@ mod tests {
         sort_individual_outputs(&mut expected_results);
         sort_individual_outputs(&mut actual_results);
 
+        let output_snap =
+            ron::ser::to_string_pretty(&expected_results, ron::ser::PrettyConfig::default()).unwrap();
+        insta::assert_snapshot!(query_name, output_snap);
```

Comment:
Out of curiosity: Why did you use this vs `insta::assert_ron_snapshot!(..)`?
This seems a bit simpler, or am I missing something?